frontend/src/redux/roadmaps/types.ts

```diff
@@ -8,6 +8,7 @@ import {
 export interface RoadmapsState {
   selectedRoadmapId?: number | null;
   taskmapPosition: TaskmapPosition | undefined;
+  fromMilestonesEditor?: string;
```

Comment:
I think the more relevant information is that this is the "back href for task overview page", which should be in the name. Currently the name doesn't really tell much, unless you already know what it is supposed to mean.

This is even more apparent when handling the other locations.